Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,22 +1,16 @@
-import React from 'react';
-import './App.css';
+import React from "react";
+import "./App.css";
+
+import HeaderContainer from "./components/HeaderComponents/HeaderContainer";
+import CardContainer from "./components/CardComponents/CardContainer";
+import Footer from "./components/FooterComponents/Footer";
 
 const App = () => {
   return (
-    <div>
-      <h3>Welcome to React Social Card!</h3>
-      <p>
-        Begin by exploring the `components` directory. You'll notice we have a
-        few files that we've already included in there to get you started right
-        away building components. You'll need to make sure you include your
-        components that you build in this file to watch your app come to life
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
+    <div className="socialCard">
+      <HeaderContainer />
```

Comment:
looks so much better when you're just building out tiny functions and passing it around instead of building everything out yourself. Great job. 🔥